test/parallel/test-v8-interceptStrings-not-Symbols.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const common = require('../common');
+const util = require('util');
+const assert = require('assert');
+
+// Note (@AnnaMag): test for unchanged behaviour after refactoring to SetHandler
+//(see: node.cc line 3158)
```

Comment:
Can you remove the references to specific lines. They will almost certainly get stale quickly.